src/common/theme.ts

```diff
@@ -137,7 +136,16 @@ export const themeOptions: ThemeOptions = {
     },
   },
 }
-
 // https://material-ui.com/customization/default-theme/#default-theme
-const theme = createTheme(themeOptions)
-export default responsiveFontSizes(theme)
+const theme: Theme = createTheme(themeOptions)
+const materialTheme = responsiveFontSizes(theme)
+const podkrepiTheme = {
+  borders: {
+    dark: colors.blue.dark,
+    light: colors.blue.main,
+    roundRadius: 60,
+  },
```

Comment:
I'm not sure why we have this here instead of in `themeOptions` :thinking:

Reply:
`ThemeOptions` are a set type by MUI. I tried putting it there, but it requires a lot of gymnastics with typings and it's not worth the hassle
![image](https://user-images.githubusercontent.com/61479393/166100346-97185bea-e7e3-44e1-91d2-fae9909e0c6f.png)

Reply:
Alright, sounds reasonable, thanks for the explanation :+1: